datasette/static/app.css

```diff
@@ -452,6 +452,10 @@ table a:link {
     margin-left: -10%;
     font-size: 0.8em;
 }
+.rows-and-columns td ol,ul {
+    list-style: initial;
+    list-style-position: inside;
```

Comment:
This is needed otherwise the bullet point marker appears in the neighbouring cell to the left, rather than the cell with the bullet point content.